src/modules/partners/partner_http.ts

```diff
@@ -6,6 +6,6 @@ import { Partner as Log } from './partner_log';
 const router = Router();
 
 router.get('/v1/partners', cache(), Log.findAll(), Handler.findAll)
-router.get('/v1/partners/suggestion', Handler.suggestion)
+router.get('/v1/partners/suggestion', cache(), Handler.suggestion)
```

Comment:
i am so curious, why we need this `cache()`, @ayocodingit ?

Reply:
when do we need cache? when the endpoint is public, the data that is rarely changed.😁